SECURITY.md

```diff
@@ -0,0 +1,123 @@
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the Rez Project -->
+
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you think you've found a potential vulnerability in rez, please
+report it by filing a GitHub [security
+advisory](https://github.com/AcademySoftwareFoundation/rez/security/advisories/new). Alternatively,
+email security@rez-project.io and provide your contact info for further
```

Comment:
Do we have this email address setup already?

Reply:
Set up! 👍